assets/cluster-bootstrap/cluster-infrastructure-02-config.yaml

```diff
@@ -11,6 +11,6 @@ status:
   apiServerURL: https://{{ .ExternalAPIDNSName }}:{{ .ExternalAPIPort }}
   etcdDiscoveryDomain: {{ .BaseDomain }}
   infrastructureName: kubernetes
-  platform: None
+  platform: IKS
```

Comment:
make platform a variable. See https://github.com/openshift/hypershift-toolkit/blob/master/cluster.yaml.example for examples.

Reply:
this is just for testing so trying to make as small amount of changes as possible and shouldn't merge this PR